game/src/game.ts

```diff
@@ -64,6 +65,7 @@ export class Game {
       decks: [{ cells: [] }, { cells: [] }],
     };
     this.cards = {};
+    this.players = [0, 1];
```

Comment:
You mentioned wanting to store whether players were AI or human, but this is just `[0, 1]` ? Are we ever going to have more than two players?